ts/features/wallet/onboarding/common/searchBank/SearchBankComponent.tsx

```diff
@@ -70,24 +69,22 @@ export const SearchBankComponent: React.FunctionComponent<Props> = (
 
   return (
     <>
-      {!props.isLoading && (
-        <Label color={"bluegrey"}>{I18n.t("wallet.searchAbi.bankName")}</Label>
-      )}
-      <Item>
-        <Input
-          value={searchText}
-          autoFocus={true}
-          onChangeText={handleFilter}
-          disabled={props.isLoading}
-          placeholderTextColor={IOColors.bluegreyLight}
-          placeholder={
-            props.isLoading ? I18n.t("wallet.searchAbi.loading") : undefined
-          }
-        />
-        {!props.isLoading && <IconFont name="io-search" />}
-      </Item>
+      <LabelledItem
+        type="text"
+        label={isLoading ? undefined : I18n.t("wallet.searchAbi.bankName")}
+        inputProps={{
+          value: searchText,
+          autoFocus: true,
+          onChangeText: handleFilter,
+          disabled: isLoading,
+          placeholder: isLoading
+            ? I18n.t("wallet.searchAbi.loading")
+            : undefined
+        }}
+        icon={isLoading ? undefined : "io-search"}
```

Comment:
The icon is on the left as it was originally on the right:

| Before | After|
|-------|-------|
|<img src="https://user-images.githubusercontent.com/11773070/126487712-5306e056-d447-46bb-be6f-6c18807e6b24.png" width=200>| <img src="https://user-images.githubusercontent.com/11773070/126487776-6ebd6e3e-a6ec-4429-a3d5-c46470aeacce.png" width=200>|

Reply:
globally, in-page search bars should have the "search" icon on the left, and potentially the "delete all" button on the right—so the image on the right looks okay